ts/utils/__tests__/convertDateToWordDistance.test.ts

```diff
@@ -1,55 +1,25 @@
-import { addHours, addYears, format } from "date-fns";
+import { format, subDays, subYears } from "date-fns";
 import { convertDateToWordDistance } from "../convertDateToWordDistance";
+import { formatDateAsLocal } from "../dates";
 
-describe("convertDateToWordDistance test plan", () => {
```

Comment:
I notice that you removed many test cases, is this necessary?
I think it was better to add some tests to the existing one than replacing them, what do you think?

Reply:
Didn't really notice these in the first place, when I did, I could see that these tests would always fail if executed between 0:00 AM and 2:00 AM because of the way the input Date object is instantiated.

Reply:
Ok. We should update failing tests so they will work again, not by removing them ;)

Reply:
I did actually _rewrite_ the failing tests, avoiding to do the same for the `else` case we decided to remove :)

Still waiting for codecov report on the new PR with the `else` case restored and tested.